modules/apps/transfer/internal/convert/convert.go

```diff
@@ -0,0 +1,57 @@
+package convert
+
+import (
+	"strings"
+
+	"github.com/cosmos/ibc-go/v8/modules/apps/transfer/internal/denom"
+	v1types "github.com/cosmos/ibc-go/v8/modules/apps/transfer/types"
+	v3types "github.com/cosmos/ibc-go/v8/modules/apps/transfer/types/v3"
+)
+
+// PacketDataV1ToV3 converts a v1 (ICS20-V1) packet data to a v3 (ICS20-V2) packet data.
```

Comment:
so where is this being used at the moment? or it isn't?

Reply:
it will eventually be used in the ibc module packet lifecycle methods (`OnRecvPacket`, `OnAckPacket` `OnTimeoutPacket`) as those methods take in a `channeltypes.Packet` and we need to ensure that the Packet's data is read correctly.